polls/models.py

```diff
@@ -83,3 +83,38 @@ class Annotation(models.Model):
 
     def __str__(self):
         return f"Annotation by {self.user.username}"
+
+
+class ExperimentType(models.Model):
+    type_choices = (
+        ("2AFC", "2AFC"),
+        ("A/B", "A/B"),
+    )
+    type = models.CharField(max_length=10, choices=type_choices)
```

Comment:
is this a hard DB constraint that can't be modified at runtime through admin UI ? why?

Reply:
We change "type" of the Experiement to a CharField with max_length=100. I guess that will do the job.